fsspec/implementations/http.py

```diff
@@ -251,7 +251,7 @@ async def _get_file(
             if isfilelike(lpath):
                 outfile = lpath
             else:
-                outfile = open(lpath, "wb")
+                outfile = open(lpath, "wb")  # noqa: ASYNC101
```

Comment:
What would be the problem here?

Reply:
https://docs.astral.sh/ruff/rules/open-sleep-or-subprocess-in-async-function/ Not sure there is a good way around it without relying the aiofiles dependency though so I just suppressed it

Reply:
What a strange rule! Just about any normal function call blocks for some time, and `open()` ought to be fast. I don't think there's a big reason to switch to `aiofiles` right now, maybe later :)

Reply:
Yeah, I just left it because it seemed useful and we can noqa the nonsense suggestions like this. 👍